packages/fxa-settings/src/components/ThirdPartyAuth/index.tsx

```diff
@@ -200,23 +163,29 @@ const ThirdPartySignInForm = ({
   );
 };
 
+function deleteParams(searchParams: URLSearchParams, paramsToDelete: string[]) {
+  paramsToDelete.forEach((param) => searchParams.delete(param));
+  return searchParams;
+}
+
 function getState() {
   // We stash originating location in the state oauth param
   // because we will need it to use it to log the user into FxA
-  const currentParams = new URLSearchParams(window.location.search);
-  currentParams.delete('deeplink');
-  currentParams.set('showReactApp', 'true');
+  const originParams = new URLSearchParams(window.location.search);
+  const modifiedParams = deleteParams(originParams, [
```

Comment:
It is probably worth a comment here why we are removing these values, which is to ensure that the user is taken to Backbone verson after they auth with third party.

Reply:
Currently, they are taken to backbone no matter what based on our router setup - IIUC, the state here is only stashed to reconstruct the redirect URL for RP. We don't read from this stashed state to decide between backbone/react on return from Google/Apple. Please correct me if I'm wrong!